torch/onnx/symbolic_opset10.py

```diff
@@ -322,6 +322,37 @@ def isfinite(g, input):
     return __not_(g, __or_(g, inf_node, nan_node))
 
 
+@parse_args("v", "f", "f", "f")
```

Comment:
use `t` instead of `v` for the first arg to enforce / express that it's a tensor value

Reply:
Based on most other symbolics, seems like parse_args("v") is used for input tensors

Reply:
Hmm I wonder what "t" is good for. OK to leave this for now.